charts/fission-all/templates/dashboards/dashboards.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.grafana.dashboards.enable }}
+{{- $files := .Files }}
+{{- range $path, $bytes := .Files.Glob "dashboards/*.json" }}
+{{- $filename := trimSuffix (ext $path) (base $path) }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ printf "%s-%s" $.Chart.Name $filename | trunc 63 | trimSuffix "-" }}
+  namespace: {{ $.grafana.Namespace }}
+  labels:
+    grafana_dashboard: "1"
```

Comment:
Should we add fission reference here ? any corrections needed here ?

Reply:
Thats the default label used by the grafana sidecar in kube prometheus stack to query for dashboards. I can change it to something like this https://github.com/prometheus-community/helm-charts/blob/cb31a4d4ca79ea92aa0628050111efeada5f7df9/charts/kube-prometheus-stack/templates/grafana/configmap-dashboards.yaml#L15 and add another label like `created-by:fission`

Reply:
yes this also looks also good. 
Additionally we can add `created-by: fission`